pyonfx/shape.py

```diff
@@ -845,12 +850,12 @@ def heart(size: float, offset: float = 0) -> Shape:
         # Build shape from template
         shape = Shape(
             "m 15 30 b 27 22 30 18 30 14 30 8 22 0 15 10 8 0 0 8 0 14 0 18 3 22 15 30"
-        ).map(lambda x, y: (x * mult, y * mult))
+        ).map(lambda x, y, typ: (x * mult, y * mult))
```

Comment:
Same as before. In general, don't force the user to define the additional parameter "typ" if it is not used.